haproxy-2.9.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: haproxy-2.9
-  version: 2.9.7
-  epoch: 1
+  version: 3.0.0
```

Comment:
this is a version streamed package yet the bump is going to v3.  The update block below should use github not release monitoring and a filter for `2.9*`

Reply:
Since it is a version streamed package, I guess we do not need to maintain package `haproxy` & package `haproxy3`?
Sadly, they do not maintain tags of minor versions of haproxy in their mirror repository on GitHub (https://github.com/haproxy/haproxy/tags), and there is no convenient way to fetch tags from a third-party git service.
I would rather keep on the release monitoring in favor of writing another _custom_ rule for fetching the latest version of haproxy.